includes/class-gdpr-audit-log.php

```diff
@@ -148,8 +151,26 @@ private static function email_mask( $email, $character = '-' ) {
 	 * @param  string $token   The 6 digit token the user gets on deletion.
 	 */
 	public static function export_log( $user_id, $token ) {
-		// see : https://github.com/trewknowledge/GDPR/blob/2.1.0/includes/class-gdpr-audit-log.php#L147.
-		throw new \DomainException('This function is not implemented.');
-	}
+		if (defined('VIP_GO_ENV') & false !== VIP_GO_ENV) {
+			// TODO: Fix this to work on VIP environments.
+			throw new \DomainException('This function is not implemented.');
+		}
```

Comment:
Will leave this for now but would like to test it out after as I think the code below will work for VIP envs as it provides a stream wrapper but want to just fix the issue at hand first then can revisit it. Need to test on a VIP env though so slow process.